Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/1-index](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/footertext](https://hosted.weblate.org/projects/jamulus/footertext/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/general](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

pljones commented on _translator-files/po/fr/Include-Server-Commands.po:
```diff
@@ -30,17 +30,15 @@ msgstr "`-e` ou `--directoryserver`  Enregistrer le serveur sur un annuaire (par
+#, no-wrap
 msgid "`-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only.\n"
-msgstr "`-f` ou `--listfilter`     Liste blanche des serveurs s'enregistrant sur la liste des serveurs, format `ip address 1[;ip address 2]` Annuaires uniquement. \n"
+msgstr "`-f` ou `--listfilter`     Liste blanche des serveurs s'enregistrant sur la liste des serveurs, format `ip address 1[;ip address 2]` Annuaires uniquement.\n"
```
Shouldn't this have the extra space before `ou`?

trebmuh replied:
Good catch. I added it.

pljones commented on _translator-files/po/fr/Installation-for-Linux.po:
```diff
@@ -206,12 +206,12 @@ msgstr "## Tout est installé ?"
 #: ../wiki/en/Installation-for-Linux.md:64
 msgid "Take a look at the"
-msgstr "Jetez un coup d'oeil à la"
+msgstr "Jetez un coup d'œil à la"
```
Should `œ` be `&‌OElig;` or is markdown happier with plain UTF-8 characters?

pljones commented on _translator-files/po/fr/Installation-for-Linux.po:
```diff
@@ -206,12 +206,12 @@ msgstr "## Tout est installé ?"
 #: ../wiki/en/Installation-for-Linux.md:65
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```
Is adding the space between `{` and `:` correct?

trebmuh replied:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but it doesn't apply here) and I didn't corrected it. I removed it.

pljones commented on _translator-files/po/fr/Installation-for-Macintosh.po:
```diff
@@ -154,7 +153,7 @@ msgstr "Jetez un coup d'œil à la"
 #: ../wiki/en/Installation-for-Macintosh.md:42
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```
Is adding the space between `{` and `:` correct?

trebmuh replied:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but it doesn't apply here) and I didn't corrected it. I removed it.

pljones commented on _translator-files/po/fr/Installation-for-Windows.po:
```diff
@@ -284,7 +284,7 @@ msgstr "Jetez un coup d'œil à la"
 #: ../wiki/en/Installation-for-Windows.md:84
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```
Is adding the space between `{` and `:` correct?

trebmuh replied:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but it doesn't apply here) and I didn't corrected it. I removed it.

pljones commented on _translator-files/po/fr/Installation-for-iOS.po:
```diff
@@ -155,4 +155,4 @@ msgstr "Jamulus a été installé et peut être utilisé maintenant. Vous pouvez
 #: ../wiki/en/Installation-for-iOS.md:50
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```
Is adding the space between `{` and `:` correct?

trebmuh replied:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but it doesn't apply here) and I didn't corrected it. I removed it.

pljones commented:
Weblate seems very noisy...

trebmuh commented:
Thanks for the catches @pljones . All should be covered by now.

pljones commented:
Did you push the changes to Github?

ann0see commented on _translator-files/po/pt_BR/1-index.po:
```diff
@@ -3,17 +3,18 @@
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2022-09-04 18:16+0000\n"
+"Last-Translator: gbonaspetti <gbonaspetti@hotmail.com>\n"
```
@gbonaspetti Thanks for your translation! I just want to make you aware of the fact that your E-Mail is visible here, currently this can't be changed.

pljones commented on _translator-files/po/fr/Installation-for-Macintosh.po:
```diff
@@ -55,34 +55,33 @@ msgstr "# Installation sous macOS"
-#| msgid "[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
 msgid "[Download Jamulus (Universal build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\\\"
-msgstr "[Téléchargez Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
+msgstr "[Téléchargez Jamulus (construction universelle)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\\\"
```
_translator-files/po/de/Installation-for-Macintosh.po line 68 above has this untranslated.  Should they be consistent?

ann0see replied:
@trebmuh ?

ann0see replied:
I think the german translation is incorrect.

pljones commented on _translator-files/po/fr/Installation-for-Macintosh.po:
```diff
@@ -154,7 +153,7 @@ msgstr "Jetez un coup d'œil à la"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Macintosh.md:42
 msgid "[Jamulus setup page](Getting-Started){: .button}"
```
I wonder why the English isn't consistent here.  In several places, the page is called "Getting Started".  The file is "Getting Started".  Yet the text here is "Jamulus setup page".  @gilgongo any ideas?

ann0see replied:
Probably that's an outdated reference to a page we deleted and now redirect to the Getting-Started page.

ann0see replied:
Opened an issue.

pljones commented on _translator-files/po/fr/Installation-for-iOS.po:
```diff
@@ -53,7 +53,7 @@ msgstr "# Installation pour iOS"
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:14
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
```
... although maybe there are more references as "Jamulus Setup page" than "Getting Started"...

ann0see replied:
Opened an issue.

ann0see commented:
Sorry all, I might have damaged the authorship here with the squash plugin.

ignotus666 commented:
@ann0see I see you've changed the name of 'Index-1'. The settings of all the other components were tied to this one (with "source directory: weblate://Jamulus/1-index) to save resources (see [here](https://docs.weblate.org/en/latest/vcs.html#internal-urls)). But now that source directory has to be changed in every single component to point to the right name (it should now be weblate://Jamulus/Startpage).
The squash add-on was installed on Index-1 and automatically applied to all components (not sure why other add-ons won't do the same, but anyway), so it isn't necessary to install that on each individual component. The only components that don't share settings with all the others are Jamulus app and Windows installer because they're in another repo.

ignotus666 commented:
Also, I think it's best to do major changes in Weblate in between PRs (freezing the components), not while they are open.

ann0see commented:
I think I didn't rename the internal name but only the display name. So it should still internally (in the URL) say 1-index

ignotus666 commented:
Hmm, but are we sure `weblate://project/component` requires the component name or its internal link?

ann0see commented:
I mean – wouldn't it break if otherwise?

ignotus666 commented:
I suppose so.

ignotus666 commented:
@ann0see did you enable the "squash commit" add-on in all the components? Could that be why the authorship is messed up? The squash commit add-on only needs to be enabled in Index-1 (or Startpage) and it applies to all linked components.

ignotus666 commented:
Oh, it's because you set squashing per language. If we want to show the authorship we have to choose "per author" or create a customised message.

ann0see commented:
Yes, or we could enable the RFC ??? Checkbox.

trebmuh commented:
>  If we want to show the authorship...

I'd be great to show authors on the commit message.

ann0see commented:
> I'd be great to show authors on the commit message.

Yes, I'll enable the RFC checkbox, but I'd certainly not like to have my real E-Mail there.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translation: Jamulus/Directories

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translation: Jamulus/Directories

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update French translation using Weblate

Currently translated at 100.0% (18 of 18 strings)

Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/fr/

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-Macintosh

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-Macintosh

ann0see commented on _translator-files/po/de/Installation-for-Macintosh.po:
```diff
@@ -65,7 +65,7 @@ msgstr "Updatest du? Es macht Sinn die [Konfiguration](Software-Manual#sicherung
 #| msgid "[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
 msgid "[Download Jamulus (Universal build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\\\"
-msgstr "[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
+msgstr "[Download Jamulus (Universal Build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
```
@pljones the fully German translation would be "Universeller Build" or "Universelle Version". @reinhardwh what would you prefer?

ann0see commented:
Please review this PR, I'd like to merge it soon.

Pasting the co authorship here:

```
Co-authored-by: ann0see
Co-authored-by: gbonaspetti
Co-authored-by: ignotus666
Co-authored-by: trebmuh
```
